doc/nrf/installation/recommended_versions.rst

```diff
@@ -53,41 +61,37 @@ The following table shows the operating system versions that support the |NCS| t
      - Not applicable
      - Tier 1
      - Tier 1
-   * - `macOS 12`_
-     - Not applicable
-     - Tier 3
-     - Tier 3
-   * - `macOS 11`_
-     - Not applicable
-     - Tier 2
-     - Tier 2
-   * - `macOS 10.15`_
-     - Not applicable
-     - Tier 3
-     - Not supported
 
 .. os_table_end
 
+Supported operating systems for firmware
+  The firmware follows the same operating system support scheme as the tools in the table above, with the following exceptions:
+
+  * Firmware is built and tested for Linux 22.04.
+  * Firmware is built for Windows 11 and macOS 14.
```

Comment:
@karhama , can you review this section, as requested by Shantha? Thanks!

Reply:
Sorry for missing this last week.